### ISSUE TITLE
[CLASSIC] Latest Version Spell Loss Effect

### ISSUE THREAD

**Issue by kuramajssken001** [cmangos/issues#971]
**Current behavior**: 
Spell error
Talent Spell ID:13743
Talent Spell ID:13875

**Expected behavior**: 
Fix Spell 13743 and 13875

**Steps to reproduce**:
1. Cast Spell 13743 or 13875.

**Client version**: 
Classic 1.12.3

**Commit hash**: 
Latest verision

**Database version**: 
Classic-DB 1.6.5

**Operating system**: 
Windows 7 x64

**Comment by cyberium** [comment]
Please confirm its the latest core -> https://github.com/cmangos/mangos-classic/commit/ad250a3bcba14c8461da0411b9b76125f0d5f36d

**Comment by Phatcat** [comment]
@cyberium http://cmangos.net/thread-7488.html - doubt **this** was fixed in the last 10 hours sadly..

**Comment by Phatcat** [comment]
Crash confirmed and reproduced on master.

Crashfix by Kraminpouds tested and confirmed working :+1:

**Comment by Phatcat** [comment]
In all honesty, I think these issues are linked: https://github.com/cmangos/issues/issues/948

**Comment by Phatcat** [comment]
@kuramajssken001 Are you still able to reproduce this on the latest master or can I close this?

**Comment by kuramajssken001** [comment]
Ok,thanks Cat!

**Comment by shnibble** [comment]
@Phatcat looks like you resolved this. Can we close?

**Comment by Phatcat** [comment]
No, cause the solution does not fix the actual issue, which has to do with unconditioned spellcasting within some scripts.
The crash just no longer happens. This is why @kuramajssken001 changed the title (and also the content of the issue report) Here is the crash report from back when I reproduced the crash to confirm it: https://gist.github.com/Phatcat/e9bce2eab1eb76a45a4e8b0015235006

I shouldn't have said to close this in my last comment cause that wouldn't have been stupid, the hotfix I introduced (by @kraminpouds, don't forget that) is only a failsafe; which means that we are still failing, we just won't crash due to *that* type of failure anymore.
